fcl/dunefdvd/g4/standard_g4_dunevd10kt.fcl

```diff
@@ -16,6 +16,8 @@ services:
   FileCatalogMetadata:   @local::art_file_catalog_mc
 
   @table::dunefdvd_simulation_services
+  PhotonVisibilityService: @local::dune10kt_vd_1x8x14_photonvisibilityservice_ArXe 
+  # The PhotonVisibilityService is used for the hybrid model for light simulation. 
```

Comment:
This should be moved to dunecore, so that the entire service block is contained in one place.  All of the service fcls live here: https://github.com/DUNE/dunecore/tree/develop/dunecore/Utilities